bin/inbox-api

```diff
@@ -78,7 +78,7 @@ def start(port, start_syncback, enable_tracer, enable_profiler):
             inbox_config['DEBUG_PROFILING_ON'] = True
         enable_profiler_api = inbox_config.get('DEBUG_PROFILING_ON')
 
-        syncback = SyncbackService(0, 0, 1)
+        syncback = SyncbackService("0", 0, 1)
```

Comment:
because of config:
```
  "SYNCBACK_ASSIGNMENTS": {
    "0": [
      0,
      1,
      2,
      3,
      4,
      5
    ]
  },
```
The key "0" is a **string**. Before moving it to the docker - it was a **number**. And it was fine in **yaml** config. But **json** config doesn't allow that.